src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -122,20 +122,28 @@ public enum VQScoreFilteringType {GENOTYPE, SITES, NONE}
     private boolean emitADs = false;
 
     @Argument(
-            fullName = "use-vqsr-classic-scoring",
-            doc = "If true, use VQSR 'Classic' scoring (vqs Lod score). Otherwise use VQSR 'Lite' scoring (calibration_sensitivity)",
+            fullName = "use-vqsr-scoring",
+            doc = "If true, use VQSR scoring (vqs Lod score). Otherwise use VETS scoring (calibration_sensitivity)",
             optional = true
     )
-    private boolean isVQSRClassic = false;
+    private boolean isVQSR = false;
 
     @Argument(
-            fullName = "vqsr-score-filter-by-site",
-            doc = "If VQSR Score filtering is applied, it should be at a site level. Default is false",
+            fullName = "vqs-score-filter-by-site",
```

Comment:
For parameters/things that are used for both VQSR and VETS, I'm using 'VQS'

Reply:
drop that in a comment for posterity?